pkg/server/computeRecommendation.go

```diff
@@ -33,6 +31,40 @@ type Request struct {
 	RequestContext context.Context
 }
 
+type Deployment struct {
+	ContainerStatuses []map[string]string `json:"containerStatuses"`
```

Comment:
Good for current PR . But we need to define the set of states for each experiment.